src/components/look-controls.js

```diff
@@ -20,6 +20,15 @@ module.exports.Component = registerComponent('look-controls', {
     this.setupMouseControls();
     this.setupHMDControls();
     this.bindMethods();
+
+    // Enable grab cursor class on canvas.
+    var sceneEl = this.el.sceneEl;
```

Comment:
It adds cognitive load. is it this variable already defined? Is it naming the same thing I want? Where is it? Yes, the linter will catch some of the issues but the it's extra work to ammend and rename variables that could introduce new bugs. I really think there's zero value on having the variable here. Let's move it up. Let's be strict with code conventions and save the creativity and energy for features